Dockerfile

```diff
@@ -34,5 +34,18 @@ COPY ./examples ./examples
 COPY ./pyproject.toml ./pyproject.toml
 
 RUN pip install -e .
+RUN chmod 777 /vega_market_sim
 
-USER vega
\ No newline at end of file
+USER vega
+
+FROM vegasim_base AS vegasim_learning
+
+COPY ./requirements-learning.txt .
+RUN  pip install -r requirements-learning.txt
```

Comment:
More my own interest than feedback - any reason for using pip over Poetry for the docker build?

Reply:
Ultimately Poetry is just overkill in a Docker environment. You're only building the once then throwing the thing away so you can just install your python packages to system python and be done with it without the weight of venvs etc